_posts/2023-11-14-specify-file-audit-log-encoding.adoc

```diff
@@ -0,0 +1,9 @@
+---
+layout: post
+title: 'Change the default encoding of the audit log file in WildFly 29'
```

Comment:
@ivassile We updated the title on the Elytron post to remove "29" (since WildFly 30 is the latest version) and mentioned the functionality being available from WildFly 29 at the beginning of the post instead.

Would you be able to drop "29" in the title here?